packages/backend/src/index.ts

```diff
@@ -270,34 +204,28 @@ async function main() {
 
   const service = createServiceBuilder(module).loadConfig(config);
 
-  // Required routers
+  // Required core routers
   await addRouter({
-    name: 'api',
     service,
     root: '/api',
     router: apiRouter,
   });
   await addRouter({
-    name: 'healthcheck',
     service,
     root: '',
     router: await createStatusCheckRouter(appEnv),
   });
-
-  // Optional routers
   await addRouter({
-    name: 'metrics',
```

Comment:
We might want to find another way to enable / disable it (since it's not a dynamic plugin)

Reply:
I think we should always leave the `metrics` and `healthcheck` router on. Thoughts? @Zaperex @kadel @tumido?

Reply:
I think it'll make sense to leave them on. Also, when I tested it with the current showcase main branch, it seems the `metrics` and `healthcheck` routers are always on regardless of the `enabled.xxx` values for each. So I think it's safe to remove the documentation for them.

Reply:
I would expect `/metrics` and `/healthcheck` endpoints to be always available. If we want the app to be a good citizen of a greater ecosystem, this should be a no-brainer. 🙂

Reply:
Perfectly OK to have them always on !